assets/js/app.js

```diff
@@ -0,0 +1,21 @@
+var templateString = document.querySelector('main-template').innerHTML;
+
+var temp = Handlebars.compile(templateString);
+var output = template('beer');
```

Comment:
Where you have `'beer'` you need to send in a JS object not a string.

You will have a single beer object when you run `forEach` on `beerData.data`.

> **NOTE** You've named your template variable `temp` on line 3 then tried to use `template` on line 4. This will give you an `undefined is not a function` error.